backend/test_real_chinese_website.py

```
@@ -0,0 +1,249 @@
+测试真实中文网站的解析和编码处理
+"""
+import uuid
+
+from sqlmodel import Session
+
+from app.core.db import engine
+from app.models.content import ContentItem
+from app.utils.content_processors import ContentProcessorFactory
+
+
+def test_real_chinese_websites():
```

Pylint (3.3.7)

[refactor] Too many local variables (22/15)

(R0914)